Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.yml

```diff
@@ -1402,7 +1403,1400 @@ script:
   longRunning: false
   longRunningPort: false
   runonce: false
-  script: '-'
+  script: |-
```

Comment:
remove the script from here

Reply:
this is the unified file, please upload the yml alone

Reply:
done